daml-foundations/daml-tools/language-server-tests/src/tests/issue-reports.ts

```diff
@@ -1,182 +0,0 @@
-// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
This test broke due to the fact that we no longer emit empty sets of diagnostics. Given that this does not test anything LSP specific and we test the same codepath in daml-ghc-shake-test-ci deleting it seems like a better option than trying to fix the test suite given that we should hopefully be able to remove all of it soon.